client/src/app/videos/+video-watch/comment/video-comments.component.ts

```diff
@@ -192,11 +192,10 @@ export class VideoCommentsComponent implements OnInit, OnChanges, OnDestroy {
     return this.authService.isLoggedIn()
   }
 
-  onNearOfBottom () {
-    this.componentPagination.currentPage++
-
+  onNearOfBottom () {    
     if (hasMoreItems(this.componentPagination)) {
       this.loadMoreThreads()
+      this.componentPagination.currentPage++
```

Comment:
Should be above `this.loadMoreThreads()` no?

Reply:
I've done a pair of tests and for me makes no difference, as the 'loadMoreThreads' function does not use 'currentPage'. The important fact is to be below 'hasMoreItems'. No problem if you want to change it @Chocobozzz ;) Thanks a lot for a quick response (as always)

Reply:
`loadMoreThreads` uses `this.componentPagination.currentPage` in https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/videos/%2Bvideo-watch/comment/video-comments.component.ts#L115 and https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/videos/%2Bvideo-watch/comment/video-comment.service.ts#L59 so yes please do the modification. And thanks to you for the bug fix :)